packages/server/src/profile/profile.controller.ts

```diff
@@ -3,13 +3,27 @@ import {
   GetProfileResponse,
   UpdateProfileParams,
 } from '@koh/common';
-import { Body, Controller, Get, Patch, UseGuards } from '@nestjs/common';
+import {
+  Body,
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  Patch,
+  Post,
+  Res,
+  UploadedFile,
+  UseGuards,
+  UseInterceptors,
+} from '@nestjs/common';
+import { FileInterceptor } from '@nestjs/platform-express';
 import { pick } from 'lodash';
 import { Connection } from 'typeorm';
 import { JwtAuthGuard } from '../login/jwt-auth.guard';
 import { NotificationService } from '../notification/notification.service';
 import { User } from './user.decorator';
 import { UserModel } from './user.entity';
+const fs = require('fs');
```

Comment:
doing import fs from 'fs', or import { fs } from 'fs'; causes and error and I don't quite know why but I'll figure that out when I have more sleep